azurelinuxagent/common/utils/extensionprocessutil.py

```diff
@@ -99,7 +99,7 @@ def read_output(stdout, stderr):
         return format_stdout_stderr("", "Cannot read stdout/stderr: {0}".format(ustr(e)))
 
 
-def format_stdout_stderr(stdout, stderr, max_len=TELEMETRY_MESSAGE_MAX_LEN):
+def format_stdout_stderr(stdout, stderr):
```

Comment:
Lesson to learn: don't assign constants in the function signature since this is loaded before the function body, so patching the constant will not work.